plugins/woocommerce-admin/client/wp-admin-scripts/print-shipping-label-banner/shipping-banner/index.js

```diff
@@ -107,7 +111,13 @@ export class ShippingBanner extends Component {
 			return;
 		}
 
-		this.acceptTosAndGetWCSAssets();
+		if ( ! activePlugins.includes( wcsPluginSlug ) && isWcstCompatible ) {
```

Comment:
Since we check for compatibility and active plugins before enqueuing the script, can we perhaps simplify the JS by assuming it's OK to show the banner?

Follow-up: I see that we use the compatibility value to determine what message to display, so this might not be entirely possible.